Makefile

```diff
@@ -6,6 +6,15 @@ TEST_CONTAINER_NAME = pg_auto_failover-test
 DOCKER_RUN_OPTS = --cap-add=SYS_ADMIN --cap-add=NET_ADMIN -ti --rm
 PDF = ./docs/_build/latex/pg_auto_failover.pdf
 
+# TEST indicates the testfile to run
```

Comment:
Like the feature. wonder if using lowercase name would be nicer

Reply:
The way make works, the variable can be set in the environment or just for the make command. That's why I believe it's usually uppercase.

```
$ TEST=test_auth make run-test
```